src/main/java/com/flowingcode/vaadin/addons/demo/TabbedDemo.java

```diff
@@ -192,8 +190,10 @@ public void showRouterLayoutContent(HasElement content) {
       }
       content = new SplitLayoutDemo(demo, sourceCodeUrl);
       currentLayout = (SplitLayoutDemo) content;
-      setSourceVisible(sourceVisible);
-      setOrientation(orientation);
+      if (splitOrientation != null) {
+        setOrientation(splitOrientation);
+        updateSplitterPosition();
```

Comment:
This is fine, as long as `codeCB` is synchronized with `setSourceVisible` (otherwise `setSourceVisible` works, but changes are lost after refreshing the content)